keel-core/src/main/kotlin/com/netflix/spinnaker/keel/core/api/ArtifactSummary.kt

```diff
@@ -4,19 +4,24 @@ import com.fasterxml.jackson.annotation.JsonIgnore
 import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.annotation.JsonInclude.Include
 import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.annotation.JsonPropertyOrder
 import com.netflix.spinnaker.keel.api.artifacts.ArtifactType
+import com.netflix.spinnaker.keel.constraints.ConstraintStateAttributes
+import com.netflix.spinnaker.keel.constraints.ConstraintStatus
 import java.time.Instant
 
 /**
  * Summarized data about a specific artifact, mostly for use by the UI.
  */
+@JsonPropertyOrder(value = ["name", "type", "versions"])
```

Comment:
Thoughts on just using `@JsonPropertyOrder(alphabetic = true)` so that this doesn't have to be updated all the time?

Reply:
I wanted names to always be on top because it's freaking annoying to find those in YAML down a bunch of other keys, especially when there's a lot of data. I don't mind keeping these up-to-date, and I don't expect the classes to change that frequently either.

Reply:
okay! Makes sense. That is very annoying.